var/spack/repos/builtin/packages/trinity/package.py

```diff
@@ -44,7 +44,7 @@ class Trinity(MakefilePackage):
 
     version('2.6.6', 'b7472e98ab36655a6d9296d965471a56')
 
-    depends_on("jdk@8")
+    depends_on("java@8:")
```

Comment:
`java@8` might actually work, that's how the `provides` command is written. Does this package only work with Java 8 or will it work with Java 9 too?

Reply:
I have not tested with java 9, but assume it will probably work